packages/esm-patient-registration-app/src/patient-registration/input/basic-input/select/select-input.component.tsx

```diff
@@ -17,7 +17,7 @@ export const SelectInput: React.FC<SelectInputProps> = ({ name, options, label }
   ];
 
   return (
-    <div>
+    <div style={{ maxWidth: '360px' }}>
```

Comment:
Belongs to CSS.

The max-width should rather be 100%. The width may be 360px.

Reply:
Ohh okay, got it! I'll push the changes.